src/databricks/labs/ucx/contexts/workflow_task.py

```diff
@@ -64,31 +64,32 @@ def submit_runs_crawler(self):
         )
 
     @cached_property
-    def clusters_crawler(self):
+    def clusters_crawler(self) -> ClustersCrawler:
         return ClustersCrawler(self.workspace_client, self.sql_backend, self.inventory_database)
 
     @cached_property
-    def pipelines_crawler(self):
+    def pipelines_crawler(self) -> PipelinesCrawler:
         return PipelinesCrawler(self.workspace_client, self.sql_backend, self.inventory_database)
 
     @cached_property
-    def table_size_crawler(self):
+    def table_size_crawler(self) -> TableSizeCrawler:
         return TableSizeCrawler(self.sql_backend, self.inventory_database, self.config.include_databases)
 
     @cached_property
-    def policies_crawler(self):
+    def policies_crawler(self) -> PoliciesCrawler:
         return PoliciesCrawler(self.workspace_client, self.sql_backend, self.inventory_database)
 
     @cached_property
-    def global_init_scripts_crawler(self):
+    def global_init_scripts_crawler(self) -> GlobalInitScriptCrawler:
         return GlobalInitScriptCrawler(self.workspace_client, self.sql_backend, self.inventory_database)
 
     @cached_property
     def tables_crawler(self):
+        # TODO: Update tables crawler inheritance to specify return type hint
```

Comment:
@pritishpai : Could you solve this?

Reply:
https://github.com/databrickslabs/ucx/pull/2819